### PR TITLE
Added Near lake tutorial for Python

### PR DESCRIPTION
**Pull request by frolvanya**

### CODE REVIEW

File: src/components/ProgrammingLanguage/index.js [hunk 1]
```diff
@@ -12,6 +13,7 @@ const LANG_MAP = {
   'js': 'JavaScript',
   'ts': 'JavaScript',
   'typescript': 'JavaScript',
+  'python': 'Python',
```
Comment (khorolets) [inline src/components/ProgrammingLanguage/index.js:16]: You've already added Python to the lang map in #19, haven't you?

File: tutorials/lake/python-nft-indexer.mdx [hunk 2]
```diff
@@ -0,0 +1,499 @@
+---
+sidebar_position: 7
+sidebar_label: NFT indexer for Python
```
Comment (khorolets) [inline tutorials/lake/python-nft-indexer.mdx:3]: I thought we've published this already. What's going on here O_o?

**Review by khorolets — CHANGES_REQUESTED** [review]
I believe you need to rebase your copy from `main` and update this PR

**Comment by frolvanya** [comment]
@khorolets Sorry for that, I forgot to rebase my fork. Now everything should be fine

**Review by khorolets — APPROVED** [review]

**Pull request merged**